ember.md

```diff
@@ -99,8 +99,14 @@ CSS is permitted (and encouraged) in apps and addons under certain circumstances
  - e.g., `display:flex`, `flex-direction: column`
 - Animations and Transitions
 
-#### Examples of Properties to Not use
-- Colors
+
+**Use discretion when adding colors to apps/addons**
+* Neutral/non-branding colors that aren't likely to change across websites are OKAY to add to that app's/addon's app styles
+* Font family and line height are NOT okay to update in the app/addon styles
```

Comment:
I agree with `font-family` coming exclusively from the host site CSS, but there are cases where `line-height` might be specific to the app/addon.